debian/files

```diff
@@ -0,0 +1 @@
+python-python-ev3dev_0.2.2.post1-1_amd64.deb python optional
```

Comment:
This file (`debian/files`) is automatically generated and should not be committed.